### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by mirronake** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#134] [Q&A] [answered]
Is it possible to use the API to see which map the player played on and how long he stayed on it?

**Comment by LeleDerGrasshalmi** [comment]
What do you mean with "Map"

**Reply by mirronake** [reply]
Fortnite Creative, to see which maps the user played on and how much he played on them.

**Reply by LeleDerGrasshalmi** [reply, marked as answer]
You cannot see that type of data

**Comment by mirronake** [comment]
and another question, how to get friends status ( like In the Launcher, Offline etc. )

![image](https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/assets/96698262/e99d6f46-639f-4cb0-9509-decab3d51393)

**Reply by LeleDerGrasshalmi** [reply]
This is handled vis XMPP.

**Comment by mirronake** [comment]
can you delete this thread.
